container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go

```diff
@@ -181,19 +186,36 @@ func (backend *DockerKurtosisBackend) GetEnclaves(
 
 	result := map[enclave.EnclaveUUID]*enclave.Enclave{}
 	for enclaveUuid, matchingNetworkInfo := range allMatchingNetworkInfo {
-
+		productionMode := false
 		creationTime, err := getEnclaveCreationTimeFromNetwork(matchingNetworkInfo.dockerNetwork)
 		if err != nil {
 			return nil, stacktrace.Propagate(err, "An error occurred getting the enclave's creation time value from enclave's Docker network '%+v'", matchingNetworkInfo.dockerNetwork)
 		}
 
 		enclaveName := getEnclaveNameFromNetwork(matchingNetworkInfo.dockerNetwork)
 
-		result[enclaveUuid] = enclave.NewEnclave(
+		// This method just looks for api-container ( which only can be one) for an enclave
+		// and extracts out whether enclave is running on production mode
+		for _, container := range matchingNetworkInfo.containers {
+			labels := container.GetLabels()
+			containerType := labels[labelTypeName]
```

Comment:
you can check if the value exists in the map: `containerType, ok := labels[labelTypeName]`